### PR TITLE
Add BERLIN to asset list

### PR DESCRIPTION
**Pull request by sirnicolaz**
This PR adds the BERLIN token, a native EVMOS ERC20 token, to the asset information in the EVMOS chain assetlist.json.

### CODE REVIEW

File: evmos/assetlist.json [hunk 1]
```diff
@@ -100,6 +100,33 @@
...
+      "display": "berlin",
+      "symbol": "BERLIN",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/evmos/images/berlin.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/evmos/images/berlin.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/evmos/images/berlin.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/evmos/images/berlin.svg"
```
Comment (JeremyParish69) [inline evmos/assetlist.json:127]: Please remove the extra padding from the svg image. (should be square dimensions)

File: evmos/assetlist.json [hunk 2]
```diff
@@ -100,6 +100,33 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/evmos/images/neok.svg"
         }
       ]
+    },
+    {
+      "description": "The token of Teledisko DAO.",
+      "denom_units": [
```
Comment (JeremyParish69) [inline evmos/assetlist.json:106]: Possible to add type.asset and address?

**Comment by JeremyParish69** [comment]
Why adding the crowdp images?

File: evmos/assetlist.json (hunk 2 above)
Comment (sirnicolaz) [reply evmos/assetlist.json:106]: I looked at the examples and couldn't find an explanation about these fields. Is it correct to assume that `type.asset` should be `erc20` and and `address` just `0x1cFc8f1FE8D5668BAFF2724547EcDbd6f013a280` ?

Comment (JeremyParish69) [reply evmos/assetlist.json:106]: Yes, I would assume so

**Comment by sirnicolaz** [comment]
@JeremyParish69 I have applied all changes you requested. let me know if there is anything else needed

**Pull request merged**